### ISSUE TITLE
Remove MiSTer-specific directories from paths in Mister Add-on packs

### ISSUE THREAD

**Issue by JosVerheij** [SmokeMonsterPacks/EverDrive-Packs-Lists-Database#697]
Currently all MiSTer Add-on packs contain the following structure in the path of all items: e.g. `[PACK_NAME]/Games/SNES/[CONTENT]`. 

Meanwhile, non-MiSTer packs do not have this structure, they omit the MiSTer specific directory placement and just follow this structure: `[PACK_NAME]/[CONTENT]`.

This is inconsistent. I use both normal and Mister Add-on SMDBs and currently I need two different comands to use them. I understand why the decision for the current Mister directory structure was made, but I feel it is unnecessary. It is trivial to add the required directories in case you want a Mister compatible directory structure.

Simply put, it is easy to add `Games/` to `--output_directory SNES/ --drop_initial_directory`. In contrast, it is fairly difficult to move either the normal SMDB output or the Mister Add-on SMDB output to the same directories.

I'd like the paths `[PACK_NAME]/Games/[SYSTEM]/[CONTENT]` to be changed to `[PACK_NAME]/[CONTENT]` in order to make Mister-Addon SMDBs consistent with the other SMDBs.

**Comment by hotelvictorcharlie** [comment]
+1 for this. Alternatively, is there any reason for us not to just move everything into the main MiSTer pack and do away with the add-ons?
